src/_scss/pages/about/_section.scss

```diff
@@ -0,0 +1,70 @@
+.about-section-wrapper {
+    margin-bottom: rem(55);
+
+    .about-section-title {
+        h2 {
+            @include h4();
+
+            font-size: rem(47);
+            line-height: rem(59);
+            color: $color-gray;
+            font-weight: 300;
+            margin: 0 0 rem(20);
+        }
+    }
+
+    .about-subtitle {
+        h3 {
+            @include h5();
+
+            color: $color-gray;
+            padding-left: rem(3);
+            margin: 0 0 rem(5);
+        }
+    }
+
+    .about-section-content {
+        margin-bottom: rem(25);
+
+        p {
+            font-size: rem(18);
+            line-height: rem(28);
+            color: $color-gray;
+            padding-left: rem(3);
+            margin: rem(20) 0;
+        }
+
+        ul {
+            margin: rem(28) 0;
+            padding-left: rem(31);
+
+            li {
+                font-size: rem(18);
+                line-height: rem(28);
+                margin-left: rem(10);
+                margin-bottom: rem(20);
+                padding-left: rem(18);
+
+                span.about-section-list-item-title {
+                    display: block;
+                    font-weight: $font-semibold;
+                }
+            }
+        }
+
+        button {
+            display: block;
+            margin: rem(19) 0 rem(38);
+            padding: rem(16) rem(32) rem(13);
```

Comment:
Any reason why the button top padding is different from the bottom padding? It results in non-vertically centered text.

Reply:
Was matching to the mock via Inspect mode. Assuming the mock wasn't fully fleshed out, I can just remove that last `rem(13)`.